src/Locale/en/Common.json

```
@@ -192,5 +192,9 @@
   "live": "Live",
   "discharged": "Discharged",
   "archived": "Archived",
-  "no_changes_made": "No changes made"
-}
\ No newline at end of file
+  "no_changes_made": "No changes made",
+  "are_you_sure_want_to_delete": "Are you sure you want to delete {{name}}?",
+  "oxygen_information": "Oxygen Information",
+  "deleted_successfully": "{{name}} Deleted Successfully",
```

This is more appropriate right?

```suggestion
  "deleted_successfully": "{{name}} deleted successfully",
```